### ISSUE TITLE
OWIN

### ISSUE THREAD

**Issue by Micha4711** [FNCSoftware/SharedWebComponentsMef#2]
Do you plan any kind of OWIN port?

I think MefConfig.Register for example uses WebAPI2 resources like ControllerBuilder which won't work using OWIN.

**Comment by kevinkemp** [comment]
No, sorry.  I just exposed an example.  I have no plans to maintain it or add features at the moment.
